### PR TITLE
Fix validations

### PR DESCRIPTION
**Pull request by joswarmer**

### CODE REVIEW

File: packages/validation/src/validators/LionWebReferenceValidator.ts [hunk 1]
```diff
@@ -191,10 +185,9 @@ export class LionWebReferenceValidator {
                 if (childNode !== undefined) {
                     if (childNode.parent !== node.id) {
                         // TODO Check that this is already tested from the child in vaidateExistsAsChild().
-                        // this.validationResult.error(`PP Parent of ${context}.${child.containment.key}[${index}] with id "${childId}" is "${childNode.parent}", but should be "${node.id}"`);
+                        //      this.validationResult.error(`PP Parent of ${context}.${child.containment.key}[${index}] with id "${childId}" is "${childNode.parent}", but should be "${node.id}"`);
```
Comment (dslmeinte) [inline packages/validation/src/validators/LionWebReferenceValidator.ts:188]: This comment could be removed as well, right?

File: packages/validation/src/validators/SimpleFieldvalidator.ts [hunk 2]
```diff
@@ -16,19 +18,15 @@ export class SimpleFieldvalidator {
...
         if (id === null || id === undefined) {
             return;
         }
-        if (typeof id !== "string") {
-            // this.validationResult.error(`${context}: Type of id "${id}" is not a string`);
-            this.validationResult.issue(new Syntax_IdFormat_Issue(new IssueContext(context), id));
-            return;
-        }
+        const idString: string = id.toString();
         const egexp = /^[a-zA-Z0-9$_-][a-zA-Z0-9$_-]*$/;
```
Comment (dslmeinte) [inline packages/validation/src/validators/SimpleFieldvalidator.ts:27]: Why `egex` without the `r`?

File: packages/validation/src/runners/RunCheckFolder.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ for (const f of getAllFiles(folder, [])) {
             console.log("V PASSED " + f) ;
             printIssues(validator.validationResult)
         } else {
-            console.log("V FAILED " + f);
+            // console.log("V FAILED " + f);
```
Comment (dslmeinte) [inline packages/validation/src/runners/RunCheckFolder.ts:24]: Why not remove?

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: packages/validation/src/validators/SimpleFieldvalidator.ts
Comment (dslmeinte) [inline packages/validation/src/validators/SimpleFieldvalidator.ts:1]: type (I guess): `SimpleFieldValidator` (capital `V`)?

File: packages/validation/src/validators/SimpleFieldvalidator.ts (hunk 2 above)
Comment (joswarmer) [reply packages/validation/src/validators/SimpleFieldvalidator.ts:27]: typo, fixed

File: packages/validation/src/runners/RunCheckFolder.ts (hunk 3 above)
Comment (joswarmer) [reply packages/validation/src/runners/RunCheckFolder.ts:24]: Why not?  Removed !

File: packages/validation/src/validators/LionWebReferenceValidator.ts (hunk 1 above)
Comment (joswarmer) [reply packages/validation/src/validators/LionWebReferenceValidator.ts:189]: Removed

File: packages/validation/src/validators/SimpleFieldvalidator.ts
Comment (joswarmer) [reply packages/validation/src/validators/SimpleFieldvalidator.ts:1]: Typo indeed, fixed

**Pull request closed without merging**